src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -11,6 +11,8 @@
 
 public partial class ConsoleEncoding : RemoteExecutorTestBase
 {
+    public static bool ConsoleEncodingAvailable => !PlatformDetection.IsUap || PlatformDetection.IsWindows10RS3OrGreater;
```

Comment:
Don't upgrade to Insiders Fast as this will break your machine (believe me, I went through that recently...)